src/data/styles/common.scss

```diff
@@ -0,0 +1,23 @@
+.pad-l-12 {
+  margin-left: 12px !important;
+}
+
+.pad-r-5 {
```

Comment:
The previous padding class is set as `!important` and this one isn't. Helper classes like these should be of equal power so that they can be used globally without causing issues. I would suggest using `!important` as a last resort in your own code, and rely on other specificity rules whenever possible.